creusot-contracts/src/resolve.rs

```diff
@@ -15,29 +21,86 @@ pub fn resolve<T: ?Sized>(_: &T) -> bool {
     true
 }
 
-#[trusted]
-impl<T1, T2: ?Sized> Resolve for (T1, T2) {
+impl<T1, T2> Resolve for (T1, T2) {
```

Comment:
Why do your remove the `?Sized` constraint?

Reply:
Because it's not actually usable as far as I know, I don't see where you would have an unsized second component to a tuple.

Reply:
That's valid Rust code:

```rust
fn f (x: &(i32, [i32])) -> &[i32]{
    &x.1
}
```

Reply:
In that case both should be unsized no?

Reply:
Nope, only the last one is allowed to be unsized, so that the compiler can compute offset.

Reply:
huh, i don't know why I didn't that would work on tuples.